app/views/collections/show.html.erb

```diff
@@ -31,9 +31,11 @@
 <div>
   <!-- Use CSS class h2 rather than HTML h2 tag to allow header and search box to align vertically -->
   <span class="h2"><%= header_title %></span>
-  <%= render partial: 'search_form'%>
+  <%= render partial: 'search_form'%><br/><br/>
+</div>
+<div style="float: right">
+<%= link_to t('sufia.dashboard.select_works'), sufia.dashboard_files_path %> | <%= link_to t('sufia.dashboard.add_work'), sufia.new_generic_work_path %>
```

Comment:
@tingtingjh I wonder if dashboard if the correct location for the translations.  I would think sufia.collection.select_works & sufia.collection.add_work